aspnetcore/razor-components/get-started.md

```diff
@@ -5,58 +5,167 @@ description: Learn how to get started with the Razor Components framework.
 monikerRange: '>= aspnetcore-3.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 01/29/2019
+ms.date: 02/03/2019
 uid: razor-components/get-started
 ---
 # Get started with Razor Components
 
+By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.com/guardrex)
+
 [!INCLUDE[](~/includes/razor-components-preview-notice.md)]
 
-## Setup
+# [Visual Studio](#tab/visual-studio)
+
+Prerequisites:
+
+[!INCLUDE[](~/includes/net-core-prereqs-vs-3.0.md)]
+
+To create your first Razor Components project in Visual Studio:
+
+1. Select **File** > **New Project** > **Web** > **ASP.NET Core Web Application**.
+1. Make sure **.NET Core** and **ASP.NET Core 3.0** are selected at the top.
+1. Choose the **Razor Components** template and select **OK**.
+
+   ![New app dialog](https://msdnshared.blob.core.windows.net/media/2019/01/razor-components-template.png)
 
-Install the following:
+1. Press **Ctrl-F5** to run the app *without the debugger*. Running with the debugger (**F5**) isn't supported at this time.
+
+Congrats! You just ran your first Razor Components app!
+
+# [Visual Studio Code](#tab/visual-studio-code)
```

Comment:
Instead of having separate sections for VSCode and VS4Mac I think we should just stick with VS and CLI for now.